protocols/smtp/src/test/java/org/apache/james/protocols/smtp/netty/NettyProxySMTPServerTest.java

```diff
@@ -0,0 +1,135 @@
+package org.apache.james.protocols.smtp.netty;
```

Comment:
The ASF V2 license header is compulsory.

Reply:
Of course.